### ISSUE TITLE
2021.01 crashing on conversion from controllers removed in 2019.66

### ISSUE THREAD

**Issue by period** [smeighan/xLights#2463]
**Describe the bug**
xLights crashes on launch after showing the splash screen following upgrade from old version.
**To Reproduce**
I've just upgraded from xLights 2019.65 (whoops..) to the latest version (2021.01) from xlights.org and whenever I launch xLights, it simply instantly crashes with no crash log generated. The splash screen is displayed for a couple of seconds before disappearing.

xFade and xSchedule launch just fine.

I have verified that I have C++ 2015-19 redistributable installed.

As far as I'm aware, there's no crash log being made. I checked my AppData/Local/Temp directory as well as my previous xlights show directory for logs and I cannot see anything. I tried launching xlights from the command prompt, though nothing to see there either.

**Versions (please complete the following information):**
 - OS: Windows 10 64bit
 - xLights version 2021.01

**Comment by period** [comment]
Fixed by deleting SL controllers from my networks.xml file manually. I assume the conversion system didn't like them as they were removed in 2019.66.

**Comment by keithsw1111** [comment]
Without a copy of your files there is not much we can do.

**Comment by period** [comment]
Here's my original xlights_networks.xml from 2019.65
[xlights_networks_backup.txt](https://github.com/smeighan/xLights/files/5852960/xlights_networks_backup.txt)

Essentially, the issue at heart here is that the conversion system from the major controller update doesn't recognise these controllers as the type was removed in 2019.66. The presence of these controllers in the xml file is causing xlights to crash without producing any kind of log or message whenever it tries to upgrade the file and it comes across these controllers.

I fixed it myself by opening up the xml file and removing the controllers manually in a text editor, but left this issue open as a fix may be useful for the (relatively tiny) userbase of these controllers who use xlights and have tried to upgrade from the last version that supported these controllers (2019.65) to the latest version.
